pkgs/applications/networking/ipfs-cluster/test.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/peer_manager_test.go b/peer_manager_test.go
+index 521e754..cf0d777 100644
+--- a/peer_manager_test.go
++++ b/peer_manager_test.go
+@@ -76,6 +76,7 @@ func clusterAddr(c *Cluster) ma.Multiaddr {
+ }
+ 
+ func TestClustersPeerAdd(t *testing.T) {
++	t.Skip("test is disabld by nixos")
```

Comment:
Are you sure this one's necessary? I just added `doCheck = true` to my system (https://github.com/bqv/nixrc/blob/live/overlays/ipfs-cluster.nix) and they all passed, I think the update fixes them.

FYI I'm on the same commit

Reply:
It failed on my machine - I expect it's a flaky test. If you feel super strongly I can reenable it, but I expect it'll fail at some point :)